seeding.js

```diff
@@ -0,0 +1,221 @@
+var db = require('./server/db.js');
+var restaurantsJson = require('./restaurant_ID.json');
+
+
+//USERS INFO
+var randNameFirst = [
+  'BrokenTableDiner',
+  'John',
+  'Jane',
+  'Chris',
+  'Albert',
+  'Leon',
+  'Claire',
+  'Sherry',
+  'Mike',
+  'Tim',
+  'David',
+  'Brad'
+];
+
+var randNameLast = [
+  '',
+  'Smith',
+  'Doe',
+  'Redfield',
+  'Kennedy',
+  'Wesker',
+  'B',
+  'W',
+  'C',
+  'M',
+  'S',
+  'A',
+  'T',
+  'R',
+  '420',
+  '2'
+];
+
+var userNameFull = () => {
+  var first = randNameFirst[Math.floor(Math.random() * (randNameFirst.length))];
+  var last = '';
+  if (first === 'BrokenTableDiner') {
```

Comment:
Why is this a special case?

Reply:
Opentable has a default username called "OpenTableDiner" so I wanted it to look like a default user